<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAboutDlg</class>
 <widget class="QWidget" name="CAboutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="m_labelNote">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>9</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>v8.1.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelIcon">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>31</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnOk">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>чбошод</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
